platform/ios/uitest/OHHTTPStubs/OHHTTPStubs/Pods/AFNetworking/AFNetworking/AFURLConnectionOperation.m

```diff
@@ -74,7 +74,7 @@ static dispatch_queue_t url_request_operation_completion_queue() {
 typedef NSCachedURLResponse * (^AFURLConnectionOperationCacheResponseBlock)(NSURLConnection *connection, NSCachedURLResponse *cachedResponse);
 typedef NSURLRequest * (^AFURLConnectionOperationRedirectResponseBlock)(NSURLConnection *connection, NSURLRequest *request, NSURLResponse *redirectResponse);
 
-static inline NSString * AFKeyPathFromOperationState(AFOperationState state) {
+static NSString * AFKeyPathFromOperationState(AFOperationState state) {
```

Comment:
AFNetworking is a third-party library being pulled in as part of OHHTTPStubs, another third-party library. Let's avoid changes to the OHHTTPStubs directory; if it is necessary to make these changes in Objective-C code, let's contribute it upstream.